### DISCUSSION TITLE
VroidSDK 3.0 in a URP project still using standard Mtoon Shader

### DISCUSSION THREAD

**Discussion by Shinnonda** [pixiv/vroid-sdk-developers#106] [Q&A] [answered]
Hi! I just would like to know if there's any bool or toggle that controls this before i start to dig into the code to figure how to make it use URP by default when models are loaded.

**Comment by nyamadan** [comment, marked as answer]
Hi! To use URP materials, you have to call LoadVRM with `UrpVrm10MaterialDescriptorGenerator` as belows.

```cs
ModelLoader.LoadVrm(... , materialGenerator: new UrpVrm10MaterialDescriptorGenerator());
```
